flink-cdc-connect/flink-cdc-source-connectors/flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnectionUtils.java

```diff
@@ -0,0 +1,55 @@
+package io.debezium.connector.postgresql.connection;
+
+import org.apache.flink.cdc.connectors.postgres.source.offset.PostgresOffset;
+import org.apache.flink.util.FlinkRuntimeException;
+
+import io.debezium.connector.postgresql.SourceInfo;
+import io.debezium.connector.postgresql.Utils;
+import io.debezium.time.Conversions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PostgresConnectionUtils {
```

Comment:
Package `io.debezium.connector.postgresql.connection` usually contains the classes copied from debezium. 
Please move this util class to package `org.apache.flink.cdc.connectors.postgres`. Maybe this class should be merged into `PostgresOffsetUtils`.

Reply:
hi @ruanhang1993, I cannot move this function into another package because it needs to access class a protected class `ServerInfo.ReplicationSlot` from debezium.

Reply:
sorry @loserwang1024 but the class ServerInfo.ReplicationSlot is protected in package `io.debezium.connector.postgresql.connection` and the Utils class is in package io.debezium.connector.postgresql` so moving the function there won't work.